src/App.java

```diff
@@ -129,6 +129,7 @@ public void schedule(Object o) {
 	public static int height;
 
 	public void startApp() {
+		AppUI.display(new Form("Loading"));
```

Comment:
What about to display current operation inside it like `nmania` does?

Reply:
That's a good idea